lib/orderbook/MatchingEngine.ts

```diff
@@ -173,6 +174,20 @@ class MatchingEngine {
 
     return matchingResult;
   }
+
+  public removeOwnOrder = (id: string): void => {
+    this.priorityQueues.ownBuyOrders.remove(id, this.removeQueuedComparator);
+    this.priorityQueues.ownSellOrders.remove(id, this.removeQueuedComparator);
+  }
+
+  private removeQueuedComparator = (queuedOrder:any, orderID:string): Boolean => {
```

Comment:
can make static, and also use `orderId` instead of `orderID` (just because that's our usual way)